sw/device/lib/dif/dif_csrng.h

```diff
@@ -424,14 +470,26 @@ dif_csrng_result_t dif_csrng_get_cmd_interface_status(
  * This function can be used before calling `dif_csrng_generate_end()` to
  * check if there is data available to read.
  *
- * @param csrng A CSRNG handle
- * @param[out] status
+ * @param csrng A CSRNG handle.
+ * @param[out] status CSRNG output status.
  * @return The result of the operation.
  */
 DIF_WARN_UNUSED_RESULT
 dif_csrng_result_t dif_csrng_get_output_status(
     const dif_csrng_t *csrng, dif_csrng_output_status_t *status);
 
+/**
+ * Gets the working state of a CSRNG instance.
```

Comment:
From the HWIP spec:
> Also, the life cycle state must be one where the signal "lc_hw_debug_en" is asserted in order to read any internal state field.

I guess this comment should also mention this requirement, right?

Reply:
Planning to continue this discussion in lowrisc/opentitan#7140. I believe the spec will need to be updated it as we need to access the internal state in `PROD` lifecycle state.

Reply:
Sounds good, it looks like the spec needs to be updated. Resolving.